domain/main.tf

```diff
@@ -3,11 +3,12 @@ data "aws_route53_zone" "root-zone" {
 }
 
 resource "aws_route53_zone" "new-zone" {
+  # checkov:skip=CKV2_AWS_39:  Requires wider discussion around log/event ingestion before implementing. To be picked up on conclusion of DBTP-974
+  # checkov:skip=CKV2_AWS_38:  
```

Comment:
What's the reason for skipping this one? Same as above?

Reply:
Looks like I just missed adding in the comment for this one. Will add in. This one is probably another one we need an investigation/spike ticket for.

Reply:
JIRA raised and number added to comment